README.md

```diff
@@ -54,16 +54,6 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] Please make sure to add the appropriate name for the project in the README file, It will make the README file more professional.